tinyagentos/routes/providers.py

```
@@ -504,6 +506,12 @@ async def add_provider(request: Request, body: ProviderCreate):
     if proxy and proxy.is_running():
         resolved = await _resolve_backend_secrets(request.app.state, config.backends)
         await proxy.reload_config(config.backends, secrets=resolved)
+    # Invalidate the models cache so the next dialog open re-reads LiteLLM
+    # with the newly-added provider.
+    try:
+        request.app.state.litellm_models_cache_at = 0.0
+    except Exception:
+        pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)